qiskit/quantum_info/operators/operator.py

```diff
@@ -224,8 +224,14 @@ def from_circuit(cls, circuit, ignore_set_layout=False, layout=None):
                 of a layout contained in the ``circuit`` input. If specified
                 the virtual bits in the :class:`~.Layout` must be present in the
                 ``circuit`` input.
+            final_layout (Layout): If specified this kwarg can be used to represent the
+                output permutation caused by swap insertions during the routing stage
+                of the transpiler.
         Returns:
             Operator: An operator representing the input circuit
+
+        Raises:
+            KeyError: If a circuit has an empty embedded.
```

Comment:
Not sure if this phrasing is correct, seems to be missing a word.

Reply:
heh, good catch. Yeah it's missing the word layout, however this actually isn't needed anymore. I think it's leftover from an earlier revision. I'll just remove this.

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/8597/commits/a0c5521a717b1bb84d9422173f2718e3c004f258